### PR TITLE
feat: uses not selectors for radios/checkboxes to not conflict with cauldron-react (for projects using dqpl that are slowly adopting cauldron-react)

### PR DESCRIPTION
**Pull request by schne324**

### CODE REVIEW

File: lib/components/checkboxes/selector.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+'use strict';
+
+module.exports = '.dqpl-checkbox:not(.dqpl-overlay-checkbox)';
```
Comment (stephenmathieson) [inline lib/components/checkboxes/selector.js]: Mixing `import` and `module.exports` seems strange. What's the logic here?

File: lib/components/checkboxes/attributes.js [hunk 2]
```diff
@@ -4,6 +4,7 @@ import Classlist from 'classlist';
 import queryAll from '../../commons/query-all';
 import isSelected from '../../commons/is-selected';
 import getLabel from '../../commons/get-label';
+import SELECTOR from './selector';
```
Comment (stephenmathieson) [inline lib/components/checkboxes/attributes.js]: Why not do:

```js
export const selector = '.some-selector.for-something'
```

?

Comment (schne324) [reply lib/components/checkboxes/attributes.js]: seems awkward to have to `import { selector } from './attributes';` in the other files that consume this selector...whats the harm in having the selector in its own file?

Comment (schne324) [reply lib/components/checkboxes/attributes.js]: _updated pr to not have selector.js files_

**Comment by schne324** [comment]
might be useful to note that the selectors added within the `:not()` are those that match only the cauldron-react components (`.dqpl-overlay-checkbox` and `.dqpl-overlay-radio` for the checkbox component and radio component)

**Pull request merged**